designs/cluster-bundles-reference.md

```diff
@@ -0,0 +1,41 @@
+# `Cluster` reference to `Bundles`
+
+## Introduction
+
+**Problem:** we rely on naming and a default namespace to find the `Bundles` object being used by an EKS-A `Cluster`.
+
+Currently, when we create a cluster, we rename the `Bundles` object we are using to have the same name as such `Cluster`. Then, when use that same name an same namespace to retrieve it from the cluster to rebuild a `cluster.Spec` in the CLI/controller.
```

Comment:
is it my english or i can't understand this sentence

Reply:
Fixed :)